backoffice/backoffice/authors/tests/test_views.py

```diff
@@ -369,6 +369,73 @@ def test_create_author(self):
         self.assertEqual(response.json()["workflow_type"], data["workflow_type"])
         self.assertIn("id", response.json())
 
+    @pytest.mark.vcr
+    def test_get_author(self):
+        self.api_client.force_authenticate(user=self.curator)
+        data = {
+            "workflow_type": WorkflowType.AUTHOR_CREATE,
+            "status": "running",
+            "data": {
+                "name": {
+                    "value": "John, Snow",
+                },
+                "_collections": ["Authors"],
+                "$schema": "https://inspirehep.net/schemas/records/authors.json",
+            },
+        }
+        url = reverse("api:authors-list")
+        response = self.api_client.post(url, format="json", data=data)
+        self.assertEqual(response.status_code, 201)
+
+        detail_url = reverse("api:authors-detail", kwargs={"pk": response.json()["id"]})
+        detail_response = self.api_client.get(detail_url)
+        self.assertEqual(detail_response.status_code, 200)
+        self.assertEqual(detail_response.json()["data"], data["data"])
+        self.assertEqual(detail_response.json()["validation_errors"], [])
+
+    @pytest.mark.vcr
+    def test_get_non_existent_author(self):
+        self.api_client.force_authenticate(user=self.curator)
+        detail_url = reverse(
+            "api:authors-detail", kwargs={"pk": "THISISFORSURENOTANID"}
+        )
+        detail_response = self.api_client.get(detail_url)
+        self.assertEqual(detail_response.status_code, 404)
+        self.assertEqual(detail_response.json()["detail"], "Not found.")
+
+    @pytest.mark.vcr
```

Comment:
Since the application is calling itself, I don’t think we need to record a cassette unless it’s interacting with external services.

Reply:
Actually they all need the cassettes as they all in a way interact with airflow in the background (so in case its not running there will be a error message)